conf/gir-gio.toml

```diff
@@ -38,3 +38,7 @@ version = "2.44"
 name = "Gio.NativeSocketAddress"
 status = "generate"
 version = "2.46"
+
+[[object]]
+name = "Gio.SocketFamily"
```

Comment:
I think this also applies to (at least) `GSocketType` and `GSocketProtocol` in addition

Reply:
@xanathar Can you also handle these two? That way we should be fine for sockets at least :)